contracts/abstract/AccessControlledAndUpgradeable.sol

```diff
@@ -51,4 +58,14 @@ contract AccessControlledAndUpgradeableModifiers is AccessControlledAndUpgradeab
     _checkRole(ADMIN_ROLE, msg.sender);
     _;
   }
+
+  modifier emergencyMitigationOnly() virtual {
+    _checkRole(EMERGENCY_ROLE, msg.sender);
+    _;
+  }
+
+  modifier minorAdminOnly() virtual {
+    _checkRole(MINOR_ADMIN_ROLE, msg.sender);
+    _;
+  }
```

Comment:
new modifiers for the two roles

Reply:
Hmm, could just use `hasRole(MINOR_ADMIN_ROLE)` etc. Anyway, this is easier to read as separate modifiers I think.